docs/configs/docsdev.js

```diff
@@ -283,6 +283,15 @@ export default {
                         title: 'Security',
                         link: '/en-us/docs/dev/user_doc/guide/security.html',
                     },
+                    {
+                        title: 'Metrics Related',
```

Comment:
I prefer "meteics" instead of "meteics related" to keep it simple and easy to read. But i do not have strong oppoin here

```suggestion
                        title: 'Metrics',
```

Reply:
Sure, I updated it with `Metrics`.